tests/openvino/tools/calibrate.py

```diff
@@ -1027,6 +1030,13 @@ def filter_configuration(config: Config) -> Config:
     return config
 
 
+def update_config(accuracy_checker_config: Config, batch_size: int) -> None:
+    for model in accuracy_checker_config["models"]:
+        for dataset in model["datasets"]:
+            print(f"Updated batch size value to {batch_size}")
+            dataset["batch"] = batch_size
```

Comment:
Was the `batch` field supported in the `get_batch_size` method? I see only `_batch_size` and `batch_size`.

Reply:
No it was not, but the batch value is set here - 
https://github.com/openvinotoolkit/nncf/pull/2197/files#diff-072c56b2f31d19962d15ed6c600cbed720f188334bb13337ae651950b69c61efR889